src/electron/views/device-connect-view/components/device-connect-port-entry.tsx

```diff
@@ -42,13 +42,14 @@ export class DeviceConnectPortEntry extends React.Component<
     public render(): JSX.Element {
         return (
             <div className={styles.deviceConnectPortEntry}>
-                <div id={textFieldLabelId} className={styles.portNumberLabel}>
+                <label htmlFor={textFieldId} className={styles.portNumberLabel}>
                     Android device port number
-                </div>
+                </label>
                 <div className={styles.deviceConnectPortEntryBody}>
                     <MaskedTextField
+                        id={textFieldId}
                         data-automation-id={deviceConnectPortNumberFieldAutomationId}
-                        aria-labelledby={textFieldLabelId}
+                        aria-labelledby={textFieldId}
```

Comment:
Is this intentional? I think you would normally omit `aria-labelledby` if you're using `label for`; won't this cause the text field to be labelled by itself?

Reply:
Ups. Good catch. Updated.